content/millennium/r4/diagnostic/observation.md

```diff
@@ -119,6 +119,35 @@ Notes:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Retrieve by id
+
+List an individual Observation by its id:
+
+    GET /Observation/M-id
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+Note: Social History is not supported
```

Comment:
Any suggestions on where to put this @joeratt

Reply:
In the Search notes, we call them `Social history Observations`.  We may also clarify that they'll always return a 404 until later implemented?

Reply:
Changed it https://github.com/cerner/fhir.cerner.com/commit/24672c363d3fb0c41210b31b55d272dc41322a4e#diff-d42a2575832583fdd3d20dff4a54f294R134